spec/index.html

```diff
@@ -1459,6 +1460,105 @@ <h3>RDF properties</h3>
+    specification, and any relevant notes (such as applicable scope).</p>
+  <p>The policy for changes to existing entries is as follows:</p>
+  <ul>
+    <li>Entries can be marked as deprecated but not deleted.</li>
+    <li>Entries can be updated to reflect any changes to the governing specification or the notes.</li>
+  </ul>
+  <p>The custodian for all registry tables is the RDF &amp; SPARQL Working Group.</p>
+
+  <section class="informative">
+    <h3><code>rdf:</code> Namespace</h3>
+    <p>The following registry table lists all the IRIs defined in the <code>rdf:</code> namespace.</p>
+    <p>A machine-readable version of this table is available at <a 
+      href="http://www.w3.org/1999/02/22-rdf-syntax-ns">
+      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns</code></a>.</p>
+<table>
+  <thead>
+    <tr>
+      <th>IRI</th>
+      <th>Specification</th>
+      <th>Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr><td><code>rdf:Alt</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:Bag</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:Seq</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:HTML</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:JSON </code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:XMLLiteral</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:List</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:Property</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:Statement</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:nil</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:rest</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:dirLangString</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:first</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:langString</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:object</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:predicate</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:subject</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:reifies</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:type</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:value</code></td><td>This document</td><td></td></tr>
+    <tr><td><code>rdf:CompoundLiteral</code></td><td>[[JSON-LD11]]</td><td>Consideration should be given to the use of <code>rdf:dirLangString</code></td></tr>
+    <tr><td><code>rdf:direction</code></td><td>[[JSON-LD11]]</td><td>Consideration should be given to the use of <code>rdf:dirLangString</code></td></tr>
+    <tr><td><code>rdf:language</code></td><td>[[JSON-LD11]]</td><td>Consideration should be given to the use of <code>rdf:dirLangString</code></td></tr>
```

Comment:
These three might be considered archaic, if not deprecated, but need to remain as long as JSON-LD uses them.